content/html/concepts/tags/terms/q/q.md

```diff
@@ -38,4 +38,4 @@ The `<q>` tag requires an opening and closing tag wrapped around a line of text.
 </p>
 ```
 
-![Example of what the rendered text would look like](../../../../../../media/q.jpg)
+![Example of what the rendered text would look like](https://raw.githubusercontent.com/Codecademy/docs/9b981e784a4869f6e903f4be2b5c3bd0ab724424/media/q.jpg)
```

Comment:
```suggestion
![Example of what the rendered text would look like](https://github.com/Codecademy/docs/blob/main/media/q.jpg)
```

Reply:
Isn't that pointing to the page with the image as opposed to the image itself?

Reply:
This is what I'm getting when I try that:

<img width="609" alt="Screen Shot 2022-03-04 at 1 11 26 PM" src="https://user-images.githubusercontent.com/23144845/156818718-3859dfb8-8e88-490e-a916-702b3f34d968.png">

Reply:
what about: https://raw.githubusercontent.com/Codecademy/docs/main/media/q.jpg

Reply:
That works! 😄